app/com/arpnetworking/metrics/portal/scheduling/JobCoordinator.java

```diff
@@ -0,0 +1,237 @@
+            System.out.println("considering " + job);
+            final JobRef<T> ref = new JobRef.Builder<T>()
+                    .setRepositoryType(_repositoryType)
+                    .setOrganization(_organization)
+                    .setId(job.getId())
+                    .build();
+            _jobExecutorRegion.tell(
+                    new JobExecutorActor.Reload.Builder<T>()
+                            .setJobRef(ref)
+                            .setETag(job.getETag())
+                            .build(),
+                    getSelf());
+        });
+
+        // We now know that all jobs in the repo have current actors.
+        // There might still be actors which don't correspond to jobs, but that's fine:
+        //   they should self-terminate next time they execute.
+
+        _periodicMetrics.recordTimer(
+                "job_coordinator_tick_time",
+                ChronoUnit.NANOS.between(startTime, _clock.instant()),
+                Optional.of(NANOS));
+        System.out.println("done with runAntiEntropy");
+    }
+
+    @Override
+    public Receive createReceive() {
+        return receiveBuilder()
+                .match(AntiEntropyTick.class, message -> {
+
+                    LOGGER.info()
+                            .setMessage("ticking")
+                            .addData("organization", _organization)
+                            .addData("repositoryType", _repositoryType)
+                            .log();
+                    if (_currentlyExecuting) {
+                        return;
+                    }
+
+                    LOGGER.info()
+                            .setMessage("starting anti-entropy")
+                            .addData("organization", _organization)
+                            .addData("repositoryType", _repositoryType)
+                            .log();
+
+                    getContext().getSystem().scheduler().scheduleOnce(
+                            scala.concurrent.duration.Duration.Zero(),
+                            () -> {
+                                try {
+                                    runAntiEntropy();
+                                } finally {
+                                    getSelf().tell(AntiEntropyFinished.INSTANCE, getSelf());
+                                }
+                            },
+                            getContext().getSystem().dispatcher()); // TODO(spencerpearson): probably not the right dispatcher
+                })
+                .match(AntiEntropyFinished.class, message -> {
+                    _currentlyExecuting = false;
+                })
+                .build();
+    }
+
+    @Override
+    public Receive createReceiveRecover() {
+        return receiveBuilder()
+                .build();
+    }
+
+    @Override
+    public String persistenceId() {
+        return "com.arpnetworking.metrics.portal.scheduling.JobCoordinator:"
+                + "repositoryType=" + _repositoryType + "; "
```

Comment:
Are you using the class name here? If so, please call the appropriate getter for the string you want.